docs/osquery/osquery.asciidoc

```diff
@@ -0,0 +1,394 @@
+[chapter]
+[role="xpack"]
+[[osquery]]
+= Osquery
+
+Osquery augments security, compliance, and operations use cases.
+For example,
+you might use Qsquery to find
+unexpected changes in disk free space or memory usage. Or you might use it to
+improve visibility into active accounts and processes.
+
+Use Osquery UI in {kib} to search, analyze, and visualize Osquery data ingested in {es}:
+
+ * Run live queries with one or more agents
+ * View a history of past queries and their results
+ * Schedule queries to capture changes to OS state over time
+ * Save queries and build a library of queries for specific use cases
+
+The UI is powered by the Osquery Manager integration.
+For information on how to set up Osquery Manager, refer to <<manage-osquery-integrations>>.
+
+[float]
+== Required privileges
+
+To use *Osquery Manager*, you must be assigned to a role with the following privileges:
+
+* `Read` privileges for the `logs-osquery_manager.result*` index.
+* {kib} privileges for **Osquery Manager**. The `All` privilege
+enables you to run, schedule, and save queries. `Read` enables you to
+view live and scheduled query results, but you can run live queries or edit anything.
+
+[float]
+[[osquery-run-query]]
+==  Run live queries
+
+Run a query against one or more agents or policies
+and view the results. This is useful for testing your query and
```

Comment:
I'd update the sentence that starts with `This is useful...` to indicate its primary use case: to query hosts during an investigation. Something like this?

> This is useful when investigating a host or to test queries you want to schedule.